netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
+
+
+== Changed Functionality in This Release
+
+This release contains the following changed functionality:
+
+* The Login and SSO authentication types were removed. +
+If your integration is based on these authentication types, you must migrate to a Request or Token-Based authentication type.
+* Other nonfunctional changes were applied to the interface of the connector to make it more consistent with other Anypoint Connectors. +
+This means that some of the operation calls in your existing application might not initially work until some minor migration tasks are performed.
+* Unknown errors are now thrown as `ANY` rather than `UNKNOWN`. +
+Consequently, error codes `DISPOSE_ERROR`, `MAPPING_ERROR`, `CANT_READ_FILE`, and `TRIGGER` are now considered `ERROR_FAULT` errors instead of `UNKNOWN`.
 * The *Map SSO* operation was removed.
-* The *UKNOWN* error code was removed. +
-All unknown errors are now thrown under the *ERROR_FAULT* error code.
-* Many operation inputs were slightly altered, to make them more consistent with other Anypoint Connectors.
 * All exposed classes have been moved from the `org.mule.module.netsuite.extension.api` package to the `com.mulesoft.connector.netsuite.extension.api` package.
 
 [[changed_operations]]
 == Changed Operations
 
+The behavior of some operations has changed in NetSuite Connector 10.0.
+
+The following operations no longer default to `payload`:
+
 [%header%autowidth.spread]
 |===
-|Operation | Parameter | Change
-
-| Change Email a| newEmail a| No longer defaults to `payload`
-| Change Password a| newPasword a| No longer defaults to `payload`
-| Get Data Center URLs a| accountId a| No longer defaults to `payload`
-| Async Search a| content a| Is now a *primary content* parameter
-| Get Async Result a| jobId a| No longer defaults to `payload`
-| Async Add List a| recordAttributes a| Is now a *primary content* parameter
-| Async Update List a| recordAttributes a| Is now a *primary content* parameter
-| Async Upsert List a| recordAttributes a| Is now a *primary content* parameter
-| Update Invitee Status List a| inviteeStatuses a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| fields a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| filters a| Is now a *content* parameter
-| Add File a| attributes a| Is now a *content* parameter
-| Add Record a| attributes a| Is now a *content* parameter.
-| Get Custom Record a| internalId a| No longer defaults to `payload` 
-| Get Deleted Records a| deletedDate a| Is now a *content* parameter
-| Update Record a| attributes a| Is now a *content* parameter
-| Upsert Record a| attributes a| Is now a *content* parameter
+|Operation | Parameter
+
+| Change Email a| newEmail
+| Change Password a| newPasword
+| Get Data Center URLs a| accountId
+| Get Async Result a| jobId
+| Get Custom Record a| internalId 
+|===
+
+The following operations are now *primary content* and *content* parameters:
```

Comment:
```suggestion
The following operations now take `primary content` and `content` parameters:
```
But...I see only one `content` parameter value?
Should this be worded as above, that these operations now have new default parameter values?

Reply:
No, they don't have new defaults